plugin-templates/blocks/theme-faceted-search/render.php

```diff
@@ -0,0 +1,19 @@
+<?php
+/**
+ * The render callback for the create-wordpress-plugin/theme-faceted-search block.
+ *
+ * All of the parameters passed to the function where this file is being required are accessible in this scope:
+ *
+ * @phpcs:disable WordPress.NamingConventions.PrefixAllGlobals.NonPrefixedVariableFound -- File doesn't load in global scope, just appears to to PHPCS.
+ *
+ * @var array    $attributes The array of attributes for this block.
+ * @var string   $content    Rendered block output. ie. <InnerBlocks.Content />.
+ * @var WP_Block $block      The instance of the WP_Block class that represents the block being rendered.
+ *
+ * @package create-wordpress-plugin
+ */
+
+?>
+<form role="search" method="get" action="/" id="search-form">
```

Comment:
```suggestion
<form role="search" method="get" action="<?php echo esc_url( home_url() ); ?>" id="search-form">
```
I'm torn on if this is a good change or not... if we had a multisite with subfolders, then "/" wouldn't be correct, but in most cases it's fine.